src/app/collection/pages/nice-challenge/components/footer/footer.component.html

```diff
@@ -0,0 +1,61 @@
+<div class="wrapper">
+  <div class="rights-container">
+    <div class="rights-text-box">
+      <p>
+        This is the page for the NICE Challenge Project curriculum 
+        collection on CLARK 
+        (<a href="www.clark.center" aria-label="Link to CLARK home page">
+        www.clark.center</a>). While CLARK hosts the curiculum all 
+        rights are retained by NICE. Learn more at 
+        <a 
+          href="https://nice-challenge.com/"
+          aria-label="Link to NICE challenge home page"
+        >
+          https://nice-challenge.com/
+        </a>  
+      </p>
+    </div>
+  </div>
+  <div class="logo-container">
+    <img 
+      src="../../../../../../assets/images/collections/NICE_logo-wo_symbol.png" 
+      alt="NICE Logo"
+    >
+    <img 
+      src="../../../../../../assets/images/collections/NSF_logo.png"
+      alt="NSF Logo"
+    >
+    <img 
+      src="../../../../../../assets/images/collections/nccp.png"
+      alt="NSA Logo"
+    >
+  </div>
+  <div class="grant-and-media">
+    <div class="grants-text-box">
+      <p>
+        The NICE Challenge Project is a grant project managed and 
+        staffed by the 501(c)(3) non-profit University Enterprises 
+        Corporation (UEC) in partnership with California State University, 
+        San Bernardino.
+      </p>
+    </div>
+    <div class="media">
+      <img 
+        src=""
+        alt="Youtube Logo"
```

Comment:
If NICE doesn't have a certain social media we can just remove that link. We can check their website for the links.